.github/workflows/test.yml

```diff
@@ -30,17 +30,9 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: ['3.7', '3.10']
-        dbt-version: [0.18.x, 0.19.x, 1.1.x, 1.4.x]
-        exclude:
-          - python-version: '3.10'
-            dbt-version: 0.18.x
-          - python-version: '3.10'
-            dbt-version: 0.19.x
-          - python-version: '3.7'
-            dbt-version: 1.1.x
-          - python-version: '3.7'
-            dbt-version: 1.4.x
+        python-version: ['3.8', '3.10']
```

Comment:
is there a reason you want to explicitly test 3.8 and 3.11?

Reply:
For testing purposes, I assuming that if it works on both the oldest and newest versions of python we want to support then it can work on all those in between. I would have said 3.7 and 3.11 to be in line with current dbt-core support, but dbt-duckdb requires python>=3.8 if I remember correctly.

Reply:
Adding on to my previous comment, if 3.11 causes any problems, I can always take care of it in a separate PR.